deploy/Dockerfile

```diff
@@ -17,9 +17,7 @@
 FROM scratch
 ADD out/executor /kaniko/executor
 ADD files/ca-certificates.crt /kaniko/ssl/certs/
-ADD files/docker-credential-gcr /usr/local/bin/
-ADD files/config.json /root/.docker/
-RUN ["docker-credential-gcr", "config", "--token-source=env"]
+ADD files/docker-credential-pass /usr/local/bin/
```

Comment:
Nice! We're actually only adding the gcr helper from a file here because it's still a special build we had to make to get it to run in this container. We're going to move it to a real release build soon. Would you mind using the ADD https:// format to add it without needing to check it in here?